taskcluster-spec/deploy.yml

```diff
@@ -91,6 +92,7 @@ resources:
           containers:
           - name: tools
             image: {$eval: "service.dockerImage"}
+            imagePullPolicy: Always
```

Comment:
Is this a separate thing?

Reply:
Yeah sorry. I just needed that to get it to pull the new version into my cluster since the actual tag hasn't changed. Figured it would be nice to have until we get a better solution.